apps/os/backend/worker.ts

```diff
@@ -39,6 +40,31 @@ export type Variables = {
 const app = new Hono<{ Bindings: CloudflareEnv; Variables: Variables }>();
 app.use(contextStorage());
 
+// Error tracking with PostHog
+app.onError((err, c) => {
+  const posthog = new PostHog(c.env.POSTHOG_PUBLIC_KEY, {
+    host: "https://eu.i.posthog.com",
+  });
+
+  // Get user ID if available
+  const userId = c.get("session")?.user?.id || "anonymous";
+
+  posthog.captureException(err, userId, {
```

Comment:
Not sure about the semantics of `app.onError`, but should this here maybe be inside `waitUntil()`?